postgis/postgis.sql.in

```diff
@@ -802,7 +808,8 @@ CREATE OPERATOR CLASS gist_geometry_ops_2d
 	FUNCTION        4        geometry_gist_decompress_2d (internal),
 	FUNCTION        5        geometry_gist_penalty_2d (internal, internal, internal),
 	FUNCTION        6        geometry_gist_picksplit_2d (internal, internal),
-	FUNCTION        7        geometry_gist_same_2d (geom1 geometry, geom2 geometry, internal);
+	FUNCTION        7        geometry_gist_same_2d (geom1 geometry, geom2 geometry, internal),
+	FUNCTION        11       geometry_gist_sortsupport_2d (internal);
```

Comment:
a useful ND sort will be to sort by hilbert(x,y) then by z, m then tiebreaking just by binary representation of the box.

Reply:
I plan to add sort support function which accepts `GIDX` as a multi-dimentional bounding box in the `gist_geometry_ops_nd`